src/test/java/org/openrewrite/kotlin/AssertionsTest.java

```diff
@@ -58,4 +59,9 @@ class A {
           )
         );
     }
+
+    @Test
+    void disablePublishingSnapshots() {
```

Comment:
@knutwannheden please feel free to remove this. I don't know if there are any changes that would affect the SAAS, so I added this as a safeguard.